docs/source/generics.rst

```diff
@@ -81,23 +81,27 @@ You may wonder what happens at runtime when you index
 of ``Stack`` that returns instances of the original class on
 instantiation:
 
->>> print(Stack)
-__main__.Stack
->>> print(Stack[int])
-__main__.Stack[int]
->>> print(Stack[int]().__class__)
-__main__.Stack
-
-Note that built-in types ``list``, ``dict`` and so on do not support
-indexing in Python. This is why we have the aliases ``List``, ``Dict``
-and so on in the ``typing`` module. Indexing these aliases gives
+.. code-block:: python
+
+   >>> print(Stack)
+   __main__.Stack
+   >>> print(Stack[int])
+   __main__.Stack[int]
+   >>> print(Stack[int]().__class__)
+   __main__.Stack
+
+Note that built-in types :py:class:`list`, :py:class:`dict` and so on do not support
+indexing in Python. This is why we have the aliases :py:class:`~typing.List`, :py:class:`~typing.Dict`
+and so on in the :py:mod:`typing` module. Indexing these aliases gives
 you a class that directly inherits from the target class in Python:
```

Comment:
@gvanrossum keeping your advice in mind in this code example:

https://github.com/python/mypy/blob/9614518856674c98e513828dab790d094a660587/docs/source/generics.rst#L460-L463

I have added the `code-block` directive here as well.